src/lib_windows.cr

```diff
@@ -0,0 +1,253 @@
+@[CallConvention("X86_StdCall")]
```

Comment:
Just a quick question, have we tried without this? If we're using `--target x86_64-pc-windows-msvc` the default calling convention for *everything* should be `X86_StdCall` so it should work without.